src/libs/actions/Report.js

```diff
@@ -1580,6 +1580,63 @@ Onyx.connect({
     },
 });
 
+/**
+ * Creates an optimistic IOU reportAction
+ *
+ * @param {String} type IOUReportAction type. Can be oneOf(create, decline, cancel, pay).
+ * @param {Number} amount IOU amount in cents.
+ * @param {String} comment user comment for the IOU.
+ * @param {String} paymentType passed only for IOUReportActions with type = pay IOU. Can be oneOf(elsewhere, payPal).
```

Comment:
> paymentType passed only for IOUReportActions with type = pay IOU.

paymentType: only required if the IOUReportAction is type 'pay'. Can be oneOf(elsewhere, payPal).